pkgs/by-name/tf/tfproviderdocs/package.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "tfproviderdocs";
+  version = "0.12.1";
+
+  src = fetchFromGitHub {
+    owner = "bflad";
+    repo = "tfproviderdocs";
+    tag = "v${version}";
+    hash = "sha256-KCkohIGkh6sg/e0qBi90hMqh/XQQNCBF6Di6V2gxqak=";
+  };
+
+  vendorHash = "sha256-fSb1C2W29zF6ygiIg6iq19A4B6ensZLqyPD5MhQ5ec8=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/bflad/tfproviderdocs/version.Version=${version}"
+    "-X github.com/bflad/tfproviderdocs/version.VersionPrerelease="
+  ];
+
+  doInstallCheck = true;
+  installCheckPhase = ''
+    $out/bin/tfproviderdocs version | grep "v${version}" > /dev/null
+  '';
```

Comment:
Use [`versionCheckHook`](https://nixos.org/manual/nixpkgs/stable/#versioncheckhook) instead.

Reply:
I didn't know about that, thank you for the suggestion! :100:  I changed the package.nix to use that hook instead :+1: